Source/Particles/ShapeFactors.H

```diff
@@ -27,44 +27,54 @@ struct Compute_shape_factor
 {
     template< typename T >
     AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-    int operator()(
-        T* const sx,
-        T xmid) const
+    amrex::Real operator()(T xmid, int cell_index) const
     {
+        using namespace amrex::literals;
+        //programme principal ecrire si cest pair j=xmid sinon on ajoute 0.5
```

Comment:
Could you translate this comment in English?

Reply:
oh yes sorry, it was more of a note for me, forgot to delete it!